core/cautils/getter/testdata/MITRE.json

```
@@ -6,6 +6,7 @@
   },
   "creationTime": "",
   "description": "Testing MITRE for Kubernetes as suggested by microsoft in https://www.microsoft.com/security/blog/wp-content/uploads/2020/04/k8s-matrix.png",
+  "typeTags": ["compliance"],
```

```suggestion
  "categories": ["compliance"],
```
I think this is a more appropriate name.

---

@dwertent 
We have used the same existing terminology as used for controls: see [this](https://github.com/search?q=repo%3Akubescape%2Fopa-utils%20ControlTypeTagCompliance&type=code).

---

1. The JSON is user-facing (unlike the code)
2. I will also change the variables naming, but it is not dependant.

---

@dwertent 
1. Not sure what you mean here - if you mean that the user will eventually see the typeTags field, i'm ok with that.